docs/src/pages/demos/tables/SimpleTable.js

```diff
@@ -35,34 +41,46 @@ const rows = [
 
 function SimpleTable(props) {
   const { classes } = props;
+  const [densePadding, setDensePadding] = React.useState(false);
```

Comment:
Can we rename the file `.hooks.js` in this case?

Reply:
@mbrookes We would imagine updating our demo infrastructure to support the case where no non-hook demo is present.

Reply:
:+1: